source/titfront/src/components/Menu.tsx

```
@@ -0,0 +1,191 @@
+      {/* Menu bar. */}
+      <div className={menuBarCn}>
+        {iota(maxGroup + 1).map((group) => (
+          <div key={group} className={cn(horizontal || "flex items-center")}>
+            {children.map(
+              (item, index) =>
+                item.props.group == group && (
+                  <Fragment key={index}>
+                    <button
+                      className={index === activeItem ? activeItemCn : itemCn}
+                      onClick={() => setActiveItem(index)}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)